src/apps/frontend/components/textarea/textarea.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+interface TextAreaProps {
+  disabled: boolean;
+  error: string;
+  name: string;
+  placeholder: string;
+  rows: number;
+}
+
+const TextArea: React.FC<TextAreaProps> = ({
```

Comment:
Should this be `Textarea`? If it should be `TextArea`, the file/folder name should be `text_area`.

Reply:
yes , this should be Textarea, updated is PR